### PR TITLE
First conversion of the Test Cases to DMN 1.2 for external validation

### PR DESCRIPTION
**Pull request by SimonRinguette**
First conversion of the test cases to DMN 1.2

I am not 100% confident in the perfection of the files. This is intended for larger testing.

### CODE REVIEW

File: runners/dmn-tck-validation/src/test/java/org/omg/dmn/tck/validation/TestCasesFiles.java [hunk 1]
```diff
@@ -24,7 +24,7 @@
     static Schema testCasesSchema;
     static {
         try {
-            dmnSchema = SchemaFactory.newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI).newSchema(new File("../../TestCases/dmn.xsd"));
+            dmnSchema = SchemaFactory.newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI).newSchema(new File("../../TestCases/DMN12.xsd"));
```
Comment (tarilabs) [inline runners/dmn-tck-validation/src/test/java/org/omg/dmn/tck/validation/TestCasesFiles.java]: ok :+1:

**Comment by falko** [comment]
The diff on the GitHub website doesn't do a good job.
Check out the branch locally and run `git diff master --color-words` to see the changes done by the migration to DMN 1.2.

**Comment by falko** [comment]
I looked through the diff details and they look good to me. In my opinion, we can merge the pull request. In case we really missed anything during this review, the execution in the engines should show bugs quickly.

**Comment by tarilabs** [comment]
As discussed on the call 2018-11-30 I will merge it just after website published with the latest discussed refresh on DMNv1.1

**Comment by tarilabs** [comment]
Thanks @SimonRinguette @falko

**Comment by StrayAlien** [comment]
@SimonRinguette @falko @tarilabs.  Thank you.  A clarification if I may - was part of the 1.2 PR to remove namespaces from typeRefs?  A quick find using 'typeRef=".*\:.*"' shows a little over 500 of them in the suite - some also in the non-compliant folder.

Just wondering if I have misunderstood something in the 1.2 spec.  If not, I'm happy to put a PR in to remove them.

Greg.

**Comment by tarilabs** [comment]
@StrayAlien Greg, can you point me on master branch a link for one instance example, please? I'm not entirely sure to what you are referring to. Tnx!

**Pull request merged**